### ISSUE TITLE
Generated directive class name should have 'Directive' suffix

### ISSUE THREAD

**Issue by Blasz** [angular/angular-cli#1217]
##### Version
```
$ ng --version
(node:20286) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
angular-cli: 1.0.0-beta.6
node: 6.2.1
os: linux x64
```

##### Repro steps

The following generate command produces a directive class name without the 'Directive' suffix.

```
$ ng g directive shared/directives/wrap-if
(node:20210) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
installing directive
  create src/app/shared/directives/wrap-if.directive.spec.ts
  create src/app/shared/directives/wrap-if.directive.ts
```

src/app/shared/directives/wrap-if.directive.ts
```
import { Directive } from '@angular/core';

@Directive({
  selector: '[wrap-if]'
})
export class WrapIf {

  constructor() {}

}
```

As per the [styleguide](https://angular.io/docs/ts/latest/guide/style-guide.html#02-03),

> Do append the symbol name with the suffix that it represents.
> ...
> export class ValidationDirective

The generated class name should be `WrapIfDirective`.

**Comment by filipesilva** [comment]
@Brocco could you have a look please?

**Comment by Blasz** [comment]
The associated spec file of the directive still imports the wrong directive name:

```
/* tslint:disable:no-unused-variable */

import { TestBed, async } from '@angular/core/testing';
import { HoverEvent } from './hover-event.directive';

describe('Directive: HoverEvent', () => {
  it('should create an instance', () => {
    let directive = new HoverEventDirective();
    expect(directive).toBeTruthy();
  });
});
```

Is this better suited to a new issue or can this one be re-opened?

**Comment by hansl** [comment]
What version are you using? `ng version`

**Comment by Blasz** [comment]
```
$ ng version
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
angular-cli: 1.0.0-beta.15
node: 6.2.1
os: linux x64
```

**Comment by hansl** [comment]
Okay I'll reopen this and assign it to @brocco.

**Comment by Brocco** [comment]
This has been resolved via PR #1940 and will be included in the next release
